app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.java

```diff
@@ -725,12 +727,19 @@ public static void showPromoteShareDialog() {
 
         EventBuilder(@NonNull String category, @NonNull String method, @Nullable String object, String value) {
             telemetryEvent = TelemetryEvent.create(category, method, object, value);
-            firebaseEvent = FirebaseEvent.create(category, method, object, value);
+            if (HomeFragment.TOPSITES_PREF.equals(value)) {
```

Comment:
Is this yet another issues other than STR described in 2033? If so, I personally prefer to separate this in another commit or even another pull request to keep a record of different things.

Reply:
This is the same problem when clicking "Clear browsing data". Not only the default top site creation is too long, also the four pref in "Clear browsing data" will exceed the param key limit